style.css

```diff
@@ -401,6 +401,70 @@ body {
   flex-direction: column;
 }
 
+.overlay {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: #191765;
+  overflow-x: hidden;
+  transition: 0.5s;
+  background-image: url(./bgimages/image-bg-1.png);
+  background-position: left top;
+  background-repeat: no-repeat, no-repeat;
+}
+
+.overlay-content {
+  position: relative;
+  top: 25%;
+  width: 100%;
+  text-align: center;
+  margin-top: 30px;
+}
```

Comment:
- [ ] Please fix the below issues and make sure your design matches the Figma design.
 - Please reduce the top margin for the mobile menu.
 - Please reduce the size of mobile menu options
 - Please use the correct color for arrows.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168054013-fc285b61-f4d2-4e39-b60c-94439c688bd1.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168054056-0a88e948-aed5-4bc3-adcb-b56efe870cc6.png)